lib/tasks/thorfile.rb

```diff
@@ -113,6 +113,37 @@ def seed
       puts "[  DONE  ]"
     end
 
+    desc "welcome WELCOME_PACKAGE", "import files and projects from a specficied welcome package"
+    def welcome(welcome_pack)
+      puts "** Importing welcome package..."
+
+      Dir.mktmpdir do |tmpdir|
+        # Extract welcome package
+        FileUtils.cp welcome_pack, tmpdir
+        Dir.chdir(tmpdir) do
+          system 'tar', "--exclude='._*'", '-xzvf', File.basename(welcome_pack)
+        end
+
+        # Copy templates
+        NoteTemplate.pwd
+        Methodology.pwd
+        FileUtils.cp_r(
+          "#{tmpdir}/welcome/templates/notes/.",
+          Configuration.find_by_name("admin:paths:note_templates").value
```

Comment:
Why don't you stick to the `Configuration.paths_templates_plugins` convention here?